### PR TITLE
Added Haaretz English, Haaretz Hebrew, and The Marker to Bundle

### PR DESCRIPTION
**Pull request by suecarmol**
Phab task: [T355668](https://phabricator.wikimedia.org/T355668)
Phab task: [T355669](https://phabricator.wikimedia.org/T355669)
Phab task: [T355670](https://phabricator.wikimedia.org/T355670)

### CODE REVIEW

File: expert/ezproxy.cfg [hunk 1]
```diff
@@ -513,6 +513,18 @@ DJ future-science.com
 # Gale
 IncludeFile databases/gale.txt
 
+# Haaretz English
+# https://phabricator.wikimedia.org/T355668
+Title Haaretz English
+URL https://www.haaretz.com/
+HJ https://www.haaretz.com/
```
Comment (jsnshrmn) [inline expert/ezproxy.cfg:521]: This is just a shotgun approach to try to rewrite more javascript.
```suggestion
HJ https://www.haaretz.com/
DJ haaretz.com
```

File: expert/ezproxy.cfg [hunk 2]
```diff
@@ -513,6 +513,18 @@ DJ future-science.com
 # Gale
 IncludeFile databases/gale.txt
 
+# Haaretz English
+# https://phabricator.wikimedia.org/T355668
```
Comment (jsnshrmn) [inline expert/ezproxy.cfg:517]: Something like this should take care of a cors error I noted. Not 100% sure if ezproxy wildurl requires escaping symbols like `?`
```suggestion
# https://phabricator.wikimedia.org/T355668
AnonymousURL -OPTIONS +https://www.haaretz.com/srv/ipAcl?isProxyServer=true*
```

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
I started by just taking a look at Haaretz English. This isn't an exhaustive review, but it looks like we're not rewriting all of the urls present in javascript. I'm also seeing some cors errors. I've added some suggestions on where to start inline.

**Comment by suecarmol** [comment]
> I started by just taking a look at Haaretz English. This isn't an exhaustive review, but it looks like we're not rewriting all of the urls present in javascript. I'm also seeing some cors errors. I've added some suggestions on where to start inline.

Thanks for the help! I have pushed the suggested changes to preprod!

File: expert/ezproxy.cfg [hunk 3]
```diff
@@ -513,6 +513,20 @@ DJ future-science.com
...
+AnonymousURL -OPTIONS +https://www.haaretz.com/srv/ipAcl?isProxyServer=true*
+Title Haaretz English
+URL https://www.haaretz.com/
+HJ https://www.haaretz.com/
+DJ haaretz.com
```
Comment (jsnshrmn) [inline expert/ezproxy.cfg:522]: I'm seeing some embedded json that looks like a remaining problem. We probably need to do some find/replace
```
                    "service": {
                        "base": "https:\u002F\u002Fwww.haaretz.com",
                        "graphql": {
                            "link": "https:\u002F\u002Fwww.haaretz.com\u002Fgql",
                            "linkCluster": "http:\u002F\u002Fwww.haaretz.com\u002Fgql",
                            "ws": "wss:\u002F\u002Fwww.haaretz.com\u002Fgql"
                        },
                        "ds": "https:\u002F\u002Fservices.haaretz.com\u002Fds",
                        "sso": "https:\u002F\u002Fservices.haaretz.com\u002Fms-sso",
                        "legacySso": "https:\u002F\u002Fsso.haaretz.com\u002Fsso",
                        "image": "https:\u002F\u002Fgif.haarets.co.il\u002Fbs",
                        "fastlyImage": "https:\u002F\u002Fimg.haarets.co.il\u002Fbs",
                        "polopolyImageBaseHref": "https:\u002F\u002Fwww.haaretz.co.il",
                        "alerts": "https:\u002F\u002Fservices.haaretz.com\u002Fuser-alerts",
                        "msPurchaseUrl": "https:\u002F\u002Fms-purchase.haaretz.com",
                        "countryIp": "https:\u002F\u002Fservices.haaretz.com\u002Fip2country\u002F",
                        "confirmation": "https:\u002F\u002Fservices.haaretz.com\u002Fms-confirmation",
                        "jwt": "https:\u002F\u002Fservices.haaretz.com\u002Fms-jwt\u002F",
                        "realizeFreeCoupon": "https:\u002F\u002Fms-apps.haaretz.com\u002Fms-coupons\u002Fcoupon\u002FrealizeCoupon",
                        "fbInstantPaywall": "https:\u002F\u002Fservices.haaretz.com\u002Ffb-paywall",
                        "customerService": "https:\u002F\u002Fservices.haaretz.com\u002Fcustomer-service",
                        "userStats": "https:\u002F\u002Fservices.haaretz.com\u002Fuser-stats",
                        "msSportResults": "https:\u002F\u002Fservices.haaretz.com\u002Fsport-results",
                        "personalization": "https:\u002F\u002Fservices.haaretz.com\u002Fpersonalization",
                        "games": "https:\u002F\u002Fservices.haaretz.com\u002Fgames-backend",
                        "brightspot": "https:\u002F\u002Fwww.haaretz.com",
                        "cookieGenerator": "http:\u002F\u002Fcookie-generator.ms-app"
                    },
                    "appFQDN": "www.haaretz.com",
                    "remoteFQDN": "www.haaretz.com",
```
I would start with something like
```suggestion
DJ haaretz.com
Find https:\u002F\u002Fwww.haaretz.com
Replace https:\u002F\u002F^swww.haaretz.com^
```
You may need to do some trial and error for escaping, and do multiple directives to catch all of the entries in that object.
For more info on find/replace in ezproxy, see:
https://help.oclc.org/Library_Management/EZproxy/Configure_resources/Find_Replace

**Pull request merged**